src/v/crypto/tests/crypto_bench.cc

```diff
@@ -50,12 +52,13 @@ struct openssl_perf {
         _svc.invoke_on_all(&crypto::ossl_context_service::start).get();
     }
 
-    ~openssl_perf() {
-        _svc.stop().get();
-        _thread_worker->stop().get();
-        _thread_worker.reset();
+    ss::future<> stop() {
+        co_await _svc.stop();
+        co_await _thread_worker->stop();
     }
 
+    ~openssl_perf() = default;
```

Comment:
Is it worth a commentIs `~openssl_perf` user-defined to disable copy and move? (Same question for `~openssl_perf_test`)

Is it worth a comment?

Reply:
I mean it wasn't really my intention to disable copy and move.  I can just remove the user defined destructor.  It's left over from development.